### PR TITLE
Removed --force options and inserted again ninja

### PR DESCRIPTION
**Pull request by wouldgo**
This PR contains:
- removed --force fixing #81;
- introduced back ninja as build tool (probably, with little modifications this script bring the module to darwin os'es).

Open for reviews.

### CODE REVIEW

File: bin/build.js [hunk 1]
```diff
@@ -15,11 +15,12 @@
     , TOOLS_DIR = PROJECT_DIR + '/tools'
     , TOOLS_DEPOT_TOOLS_DIR = TOOLS_DIR + '/depot_tools'
     , GCLIENT = 'gclient'
+    , NINJA = 'ninja'
     , NODE_GYP = PROJECT_DIR + '/node_modules/node-gyp/bin/node-gyp.js'
     , VERBOSE = false;
 
   process.env.PATH = process.env.PATH + ':' + TOOLS_DEPOT_TOOLS_DIR;
-  process.env.GYP_GENERATORS = 'make';
+  process.env.GYP_GENERATORS = NINJA;
```
Comment (modeswitch) [inline bin/build.js:23]: Why did you change this back?

**Comment by wouldgo** [comment]
since I saw on my apple laptop that make complains about missing symbols. Tried with ninja (out of the building script) and works.
Plus I saw that ninja set automatically the `-j` make options (speeding the compiling process).
